About Section

chelseakari opened the pull request:
## Changes  
- Changed the `App.jsx` default route from `ComingSoon` to `Home`.  
- Moved the About section from Home into its own component at `src/app/components/about`.  
- Added a constant up-and-down animation to all images in the About section.  
- Made the Hack and WICS logo bubbles clickable links to their respective websites.  
- Adjustments for the mobile version:  
  - Centered the paragraphs.  
  - Hid the WICS and Hack logo bubbles.  
  - Reduced animation movement slightly.  

## Instructions for Running This Branch  
1. Comment out all sections other than About in `Home.jsx`.  
2. Set a background color or image in `Home.scss`. Currently, it references a deleted asset, so it won’t display properly.

IanWearsHat commented on site/src/app/views/home/Home.scss:
```diff
@@ -323,55 +323,26 @@ body {
 		flex-direction: column;
 
 		.section-title {
```
Issue: It would be better if the about heading was centered for mobile views (I added dummy text above just to visualize what it would look like if the navbar wasn't right next to the heading).

![image](https://github.com/user-attachments/assets/d10d2e4b-3b3e-43dd-8905-8cb99bf55ec9)

IanWearsHat commented on site/src/app/components/about/About.scss:
```diff
@@ -0,0 +1,157 @@
+	width: 40%;
+	font-family: "Instrument Sans", serif;
+	font-weight: 500;
+	font-size: 25px;
```
Suggestion: We would like to use rem instead of px for font sizes.

IanWearsHat reviewed (changes requested):
Nice work on this! I really like the slight animations. It really adds to the website. I have a few notes on adapting to larger screens and a few other styling changes.

chelseakari commented:
Thanks for the review, I tried addressing the mentioned issues.
1. Moved h2 into the `about-content` div in the `About.jsx` file. The about paragraph is in the same div so now they are aligned together.
2. The otter and scuba images use vh instead of vw for their heights now, so the extra space should be reduced.
3. Set max width and heights for all images so the bubbles shouldn't get too big at large window sizes now.
4. `about-content` div aligns items to center for smaller screens, so that centers the h2 as well.
5. Set the `about-text` font size to 1.56rem.

IanWearsHat commented on site/src/app/components/about/About.jsx:
```diff
@@ -0,0 +1,73 @@
+						target="_blank"
+						rel="noopener noreferrer"
+					>
+						<div class="floating-logo wics" aria-label="WICS Logo"></div>
```
Issue: `class` should be `className` in React.

IanWearsHat commented on site/src/app/components/about/About.jsx:
```diff
@@ -0,0 +1,73 @@
+						target="_blank"
+						rel="noopener noreferrer"
+					>
+						<div class="floating-logo hack" aria-label="Hack Logo"></div>
```
Issue: `class` should be `className` in React.

IanWearsHat commented on site/src/app/components/about/About.scss:
```diff
@@ -0,0 +1,188 @@
+}
+
+.wics {
+	top: calc(-170px - 1vw);
```
Issue: From around 900px to 1000px, the bubbles overlap with the heading.

![image](https://github.com/user-attachments/assets/acd0b5b6-d612-4f15-af6c-b33f679f5640)

IanWearsHat reviewed (changes requested):
Thanks for addressing my notes. I have a minor note on React prop names and one on bubble positioning.

chelseakari commented:
1. Renamed all `class` to `className` in `About.jsx`
2. Changed the logo position values and checked that it looks fine at a window width of 900~1000px and at larger screen sizes.

IanWearsHat reviewed (approved):
Great work on this! Squash and merge when ready.
